index.js

```diff
@@ -32,20 +32,20 @@ const slowDown = PCancelable.fn(async ({config, convert, exportOptions, inputPat
 });
 
 const slowDownService = {
-	title: 'Slow down',
+	title: 'Slow Down',
 	config: {
 		slowDownPercent: {
-			title: 'Slow down percent',
-			description: 'Percent by which to slow down the video by. For example, 0.5 means half speed, and double the duration.',
+			title: 'Slow Down Percentage',
+			description: 'Slow down the video by the given percentage. For example, 0.5 means half speed and double the duration.',
 			type: 'number',
 			minimum: 0,
 			maximum: 1,
 			default: 0.5,
 			required: true
 		},
 		slowDownAudio: {
-			title: 'Slow down audio',
-			description: 'This only works if slow percent is less or equal to 0.5',
+			title: 'Slow Down Audio',
+			description: 'This only works if “Slow Down Percentage” is less than or equal to 0.5',
```

Comment:
It's a bit inconsistent that the above ends in a `.`, while this does not. Should we maybe say that all descriptions should end in a dot `.`?